build-system/pr-check/performance-tests.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+/**
+ * @fileoverview
+ * This script runs performance tests.
+ * This is run during the CI stage = test; job = performance tests.
+ */
+
+const colors = require('ansi-colors');
+const {
+  downloadDistOutput,
+  printChangeSummary,
+  startTimer,
+  stopTimer,
+  timedExecOrDie: timedExecOrDieBase,
+} = require('./utils');
+const {determineBuildTargets} = require('./build-targets');
+const {isTravisPullRequestBuild} = require('../common/travis');
+
+const FILENAME = 'performance-tests.js';
+const FILELOGPREFIX = colors.bold(colors.yellow(`${FILENAME}:`));
+const timedExecOrDie = (cmd) => timedExecOrDieBase(cmd, FILENAME);
+
+async function main() {
+  const startTime = startTimer(FILENAME, FILENAME);
+
+  if (!isTravisPullRequestBuild()) {
+    downloadDistOutput(FILENAME);
+    timedExecOrDie('gulp update-packages');
+    timedExecOrDie('gulp performance --nobuild --quiet');
+  } else {
+    printChangeSummary(FILENAME);
+    const buildTargets = determineBuildTargets(FILENAME);
+    if (buildTargets.has('RUNTIME') || buildTargets.has('FLAG_CONFIG')) {
+      downloadDistOutput(FILENAME);
+      timedExecOrDie('gulp update-packages');
+      timedExecOrDie('gulp performance --nobuild --quiet');
```

Comment:
Optional: Do you think it's worth initially making this `timedExec` for a couple of days to see how these tests do on Travis?

Reply:
What's the difference if this is the only test that runs? I plan on moving this to master push builds as part of this PR.

Reply:
In that case, it's a moot point. Carry on!

Reply:
Hmm well, in order to not break master, I'll set this job to be allowed to fail.